src/components/EventCalendar/Calendars.tsx

```diff
@@ -1,7 +1,7 @@
 import EventListCard from 'components/EventListCard/EventListCard';
 import dayjs from 'dayjs';
-import React, { useState, useEffect } from 'react';
-import styles from './Calendar.module.css';
+import React, { useState } from 'react';
+import styles from './Calendars.module.css';
```

Comment:
Why was the file renamed?

Reply:
Because I kept the generic calendar name for react-calendar I integrated